README.md

```diff
@@ -24,8 +24,8 @@ For example:
 
 This extension contributes the following settings:
 
-* `myExtension.enable`: enable/disable this extension
-* `myExtension.thing`: set to `blah` to do something
+- `myExtension.enable`: enable/disable this extension
+- `myExtension.thing`: set to `blah` to do something
```

Comment:
Is this going to be shown in the extension "store" page? Maybe we can make a copy of this and remove most of the things that are placeholder?

Reply:
Oh my, too late.

Reply:
Let's do that next

Reply:
Ok I will open a new PR